src/workerd/io/actor-cache.h

```diff
@@ -239,6 +239,8 @@ class ActorCacheInterface: public ActorCacheOps {
   virtual kj::Promise<kj::String> getBookmarkForTime(kj::Date timestamp);
   virtual kj::Promise<kj::String> onNextSessionRestoreBookmark(kj::StringPtr bookmark);
   virtual kj::Promise<void> waitForBookmark(kj::StringPtr bookmark);
+
+  virtual void ensureReplicas();
```

Comment:
Given that it's just a throw, consider just inlining the implementation with the throw here. Just keeps things a bit simpler.

Reply:
https://github.com/cloudflare/workerd/compare/9e46063539971abff34e8043709d95965cbec721..999831c699fd945aaddc50c446e07a6986d4a3b2